girder/api/v1/assetstore.py

```diff
@@ -248,3 +250,73 @@ def getAssetstoreFiles(self, assetstore, limit, offset, sort, params):
         return list(self.model('file').find(
             query={'assetstoreId': assetstore['_id']},
             offset=offset, limit=limit, sort=sort))
+
+    @access.admin
+    @autoDescribeRoute(
+        Description('get space usage statistics by collection and by user.')
+        .modelParam('id', model='assetstore')
+        .errorResponse()
+        .errorResponse('You are not an administrator.', 403)
+    )
+    def getSpaceUsageStatistics(self, assetstore, params):
+        filesInStore = list(self.model('file').find(
+            query={'assetstoreId': assetstore['_id']}))
```

Comment:
This could be many thousands of files; we shouldn't pull them all into memory via `list()`. We'll want to iterate over the cursor.